wsd/Admin.cpp

```diff
@@ -348,6 +348,7 @@ void AdminSocketHandler::handleMessage(const std::vector<char> &payload)
         const std::string& dockey = tokens[2];
         const std::string& routeToken = tokens[3];
         const std::string& serverId = tokens[4];
+        const std::string& autoSaveAndStop = tokens[5];
```

Comment:
What happens where there are only 5 tokens? Why aren't we checking for the correct number of tokens (on line 345) ?

Reply:
This should be safe - the StringVector returns "" for un-defined tokens - please check; and ensure for back-compat the code paths work nicely with ""